app/assess/data.py

```diff
@@ -74,12 +74,16 @@ def get_fund(fund_id: str) -> Union[Fund, None]:
         fund_id=fund_id
     )
     response = get_data(endpoint)
-    if len(response) > 0:
-        fund = Fund.from_json(response[0])
-        if "rounds" in response and len(response["rounds"]) > 0:
-            for fund_round in response["rounds"]:
-                fund.add_round(Round.from_json(fund_round))
-        return fund
+    try:
+        if len(response) > 0:
+            fund = Fund.from_json(response[0])
+            if "rounds" in response and len(response["rounds"]) > 0:
+                for fund_round in response["rounds"]:
+                    fund.add_round(Round.from_json(fund_round))
+            return fund
+    except KeyError:
+        if type(response) == dict:
+            return response
```

Comment:
This seems weird that you have to manage two different types from an endpoint

Reply:
Seems like this endpoint only returns a dict - maybe we can remove the array specific logic?

https://github.com/communitiesuk/funding-service-design-fund-store/blob/225163028d60335102570850c212ffb8b0ab20a7/core/funds.py#L72

Might be good to add a test for this